### ISSUE TITLE
Cannot Download Kaldi Model (Enable Full Transcription)

### ISSUE THREAD

**Issue by edglazer** [lowerquality/gentle#231]
Hello,
I've been unable to test the mac app using the full transcription. Selecting "Enable Full Transcription" on the menu either crashes the program or just runs without actually downloading anything.

I try to download manually from https://lowerquality.com/gentle/kaldi-models-0.03.zip
but for me, this download fails with network error after about 3 MB. I see a reference to Daniel Povey's Fisher-English models in the wiki page here, but that just points to a a directory of 1.3GB, and its unclear which files I need to download from that repository. 

Happy to do the downloading manually, but need to know what to grab. Thanks for this great tool!

**Comment by edglazer** [comment]
Found a solution to manually download that worked for me, use curl:
`curl -OC - https://lowerquality.com/gentle/kaldi-models-0.03.zip`
Worked in one go for me, but that command should also handle restarts if needed.

**Comment by MysteryPancake** [comment]
I am having the same issue. Where did you place the file "kaldi-models-0.03.zip" to get it to recognize it?

**Comment by edglazer** [comment]
Short version: the most recent time I ran Gentle (the prebuilt MacOS binary), I clicked the download from the menu and it worked. 

Slightly longer version: There must be more going on behind the scenes because I see in line 42 of the path definition file (https://github.com/lowerquality/gentle/blob/master/gentle/util/paths.py#L42) that there will be a .gentle folder created in my home directory. Looking there (again, after successfully downloading from within the Gentle app), I see a directory structure of:
~/.gentle/exp/tdnn_7b_chain_online/graph_pp/HCLG.fst
with that HCLG.fst file being ~950 MB in size. 

When I compare that to the contents of the kaldi-models-0.03.zip file, a similar folder structure exists, but there are a bunch of files (and it looks like the total package is closer to ~200mb in size?). Not sure how gentle (probably using a kaldi binary) turns that zip package into that HCLG.fst file, but that *seems* to be the final resting place for what is downloaded in the kaldi-models-0.03.zip package.

**Comment by anarchivist** [comment]
@edglazer, I have a freshly downloaded Gentle MacOS binary, and running it for the first time, I ran into the crashing issue that you reported. I also looked at #145, and the referenced `install_language_model.sh` script.

What worked was to download the referenced language model tarball (https://lowerquality.com/gentle/aspire-hclg.tar.gz) and unpack the contents into `~/.gentle`. Full transcription now works for me.
